internal/awsservice/cloudwatchlogs.go

```diff
@@ -95,10 +95,6 @@ func DeleteLogStream(logGroupName, logStreamName string) {
 
 // DeleteLogGroup cleans up log group by name
 func DeleteLogGroup(logGroupName string) {
-	// catch ResourceNotFoundException when deleting the log group and log stream, as these
```

Comment:
Delete this since we already has the [declaration](https://github.com/aws/amazon-cloudwatch-agent-test/pull/166/files#diff-481de9e23b7d675ebe3f425909e7ce4585e30859cddc0183817e636124dd07f2R21)